app/services/linkService.js

```diff
@@ -6,10 +6,6 @@ const POST_URL = `${BASE_URL}/shorten`;
 const requestCreateShortenLink = (url) => {
   fetch(POST_URL, {
     method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
```

Comment:
For some reasons `whatwg-fetch` uses `OPTIONS` instead of `POST` when you have content-type header.

Reply:
Thanks Evgeniy.

I will resume the project and will let you know when is done.